keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -236,7 +246,10 @@ on version X send a request to the server of version less than X which does not
 some of the fields (or a similar situation). We do not think it is worth
 supporting this use case initially and will error if clients attempt to validate
 schema server-side with protobuf data.
-
+* Enhanced offline validation. Current client-side validation has been used as a
+  form of offline validation to simply validate configs that have no intention
+  of being actually applied to a server. Ideally users would have a supported
+  way to perform offline validation, but that is outside the scope of this KEP.
```

Comment:
Not asking for a change here, but using the same yaml/json library client side would already be an improvement (even if it doesn't work with openapi support, it would at least detect duplicates for example?)